### PR TITLE
Remove trad. client documentation in configuration management

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/uyuni-project/uyuni-docs/issues/2426

# Target branches

* Which documentation version does this PR apply to?
* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

- [ ] master
- [ ] manager-4.3

# Links
- Related development PR #<insert PR link, if any>
- Fixes #<insert spacewalk issue, if any>

### CODE REVIEW

File: modules/client-configuration/pages/configuration-management.adoc [hunk 1]
```diff
@@ -3,29 +3,26 @@
...
+////
 
 Configuration channels are used to organize configuration files.
 You can subscribe clients to configuration channels, and deploy configuration files as required.
 
 Configuration files are version-controlled, so you can add configuration settings, test them on your clients, and roll back to a previous revision as required.
 When you have created your configuration channels, you can also perform comparisons between various configuration files, and between revisions of the same configuration file.
 
-Configuration files can be centrally managed, or locally managed.
+Configuration files can be centrally managed.
```
Comment (raulillo82) [inline modules/client-configuration/pages/configuration-management.adoc:24]: This line is the same as the one in line 9. Maybe it is not the purpose of this PR, but as there's been a modification here, would it be possible to remove it from one of the places? Actually even the beginning of the next sentence is also above...

**Review by raulillo82 — CHANGES_REQUESTED** [review]
Other a small comment about a duplicated sentence/paragraph, everything else LGTM.

File: modules/client-configuration/pages/configuration-management.adoc (hunk 1 above)
Comment (0rnela) [inline modules/client-configuration/pages/configuration-management.adoc:24]:
```suggestion
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

**Review by 0rnela — APPROVED** [review]

File: modules/client-configuration/pages/configuration-management.adoc (hunk 1 above)
Comment (0rnela) [reply modules/client-configuration/pages/configuration-management.adoc]: Done.

File: modules/client-configuration/pages/configuration-management.adoc [hunk 2]
```diff
@@ -3,29 +3,25 @@
...
-
 Configuration parameters are scripted and stored in configuration files.
 You can write configuration files directly using the {productname} {webui}, or you can upload or link to files that exist in other locations.
 
-Configuration files can be centrally managed, or locally managed.
+Configuration files can be centrally managed.
 Centrally managed configuration files are provided by global configuration channels and can be applied to any client subscribed to the {productname} Server.
+
+////
+// 2023-10, julio:
```
Comment (keichwa) [inline modules/client-configuration/pages/configuration-management.adoc:13]:
```suggestion
// 2023-10, julio: such features are somehow covered by salt. Is dedicated documentation
// for this required?
```

File: modules/client-configuration/pages/configuration-management.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/configuration-management.adoc]: thanks for input and help!

**Pull request merged**